contrib/images/Makefile

```diff
@@ -1,20 +1,34 @@
 RELAYER_TAG := $(shell grep '^ENV RELAYER_TAG' cosmos-relayer/Dockerfile | cut -f3 -d\ )
+BABYLON_FULL_PATH := $(shell git rev-parse --show-toplevel)
+BABYLON_VERSION_BEFORE_UPGRADE ?= 5f8af8ced17d24f3f0c6172293cd37fb3d055807
```

Comment:
from this point forward we should always treat v0.9.0 as previous verison, so maybe we can start testing against https://github.com/babylonlabs-io/babylon/tree/v0.9.0 tag ?

Reply:
yeap, I was waiting for the merge 
I will update

Reply:
done at https://github.com/babylonlabs-io/babylon/pull/3/commits/416517394b3da8f748cd285af6cca0922c67d896